src/wp-admin/authorize-application.php

```diff
@@ -57,7 +57,7 @@
 
 $app_name    = ! empty( $_REQUEST['app_name'] ) ? $_REQUEST['app_name'] : '';
 $success_url = ! empty( $_REQUEST['success_url'] ) ? $_REQUEST['success_url'] : null;
-$reject_url  = ! empty( $_REQUEST['reject_url'] ) ? $_REQUEST['reject_url'] : $success_url;
+$reject_url  = ! empty( $_REQUEST['reject_url'] ) ? $_REQUEST['reject_url'] : add_query_arg( 'success', 'false', $success_url );
```

Comment:
If `$success_url` is null (as the get variable was unused) the `$reject_url` would try to add `success=false` onto null, so it would be appended to the *current* url for the reject url.  We should probably override `$reject_url` to null as well here if the fail condition of this ternary is met.

Realistically, this line probably should stop being a ternary for readability purposes.

Reply:
Ah, nice catch. How about b9111114f166d4b6dca1b437f97256ca7335cfb6.